src/status_im/native_module/impl/module.cljs

```diff
@@ -152,6 +152,30 @@
   (when (status)
     (.disableInstallation (status) installation-id callback)))
 
+(defn load-filters [chats callback]
```

Comment:
ok here it is rpc calls so the native code above is going to be removed?

Reply:
correct